File: boat-common/src/test/java/test/xyz/srclab/common/test/TestMarkerTest.java
package test.xyz.srclab.common.test;

import org.testng.annotations.Test;
import xyz.srclab.common.test.TestMarker;

/**
 * @author sunqian
 */
public class TestMarkerTest {

    @Test
    public void testTestMarker() {
        TestMarker testMarker = TestMarker.newTestMarker();
        testMarker.mark("hello");
        testMarker.mark("world", "!");
        System.out.println(testMarker);
    }
}

File: boat-common/src/test/kotlin/test/xyz/srclab/common/test/TestMarkerTest.kt
package test.xyz.srclab.common.test

import xyz.srclab.common.test.TestMarker
import kotlin.test.Test

object TestMarkerTestKt {

    @Test
    fun testTestMarker() {
        val testMarker = TestMarker.newTestMarker()
        testMarker.mark("hello")
        testMarker.mark("world", "!")
        println(testMarker)
    }
}